modules/catalog/src/test/java/org/apache/ignite/internal/catalog/CatalogManagerSelfTest.java

```diff
@@ -1040,39 +1040,11 @@ public void testCreateSortedIndex() {
         assertEquals("VAL", index.columns().get(0).name());
         assertEquals("ID", index.columns().get(1).name());
         assertEquals(CatalogColumnCollation.DESC_NULLS_FIRST, index.columns().get(0).collation());
-        assertEquals(CatalogColumnCollation.ASC_NULLS_LAST, index.columns().get(1).collation());
+        assertEquals(ASC_NULLS_LAST, index.columns().get(1).collation());
         assertTrue(index.unique());
         assertFalse(index.writeOnly());
     }
 
-    @Test
-    public void testCreateIndexWithSameName() {
-        assertThat(manager.createTable(simpleTable(TABLE_NAME)), willBe(nullValue()));
-
-        CreateHashIndexParams params = CreateHashIndexParams.builder()
-                .indexName(INDEX_NAME)
-                .tableName(TABLE_NAME)
-                .columns(List.of("VAL"))
-                .build();
-
-        assertThat(manager.createIndex(params), willBe(nullValue()));
-        assertThat(manager.createIndex(params), willThrow(IndexAlreadyExistsException.class));
-    }
-
-    @Test
-    public void testCreateIndexOnDuplicateColumns() {
```

Comment:
Why do you drop this test?
Duplicate columns looks like a typo. E.g. in "VAL1, VAL2, VAL1" we can't be sure if user means:
"VAL1, VAL2" or "VAL2, VAL1" or "VAL1, VAL2, VAL3" or whatever else. In general, this case is invalid.